### PR TITLE
fix: change order detail page route 

### PR DESCRIPTION
**Pull request by irina-sid**
The type of this PR is: **FIX**

This PR resolves [PX-4070]

### Description

Have corrected routes according to the comments to /pull/4876 (previous PR).

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[PX-4070]: https://artsyproduct.atlassian.net/browse/PX-4070

### CODE REVIEW

File: src/lib/navigation/__tests__/routes-tests.tsx [hunk 1]
```diff
@@ -1258,4 +1258,11 @@ describe(webViewRoute, () => {
       url: "/artist/banksy/auction-results/8907",
     })
   })
+
+  it("inlines params in the original order history route", () => {
+    const matcher = webViewRoute("/order-history/:orderID")
+    expect(matcher.match(["order-history", "8907"])).toEqual({
+      url: "/order-history/8907",
```
Comment (MounirDhahri) [inline src/lib/navigation/__tests__/routes-tests.tsx:1265]: **issue:** I think you missed to update the test here

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thanks for making these changes @irina-sid. I left a minor comment and we can then merge this

File: src/lib/navigation/__tests__/routes-tests.tsx (hunk 1 above)
Comment (irina-sid) [reply src/lib/navigation/__tests__/routes-tests.tsx:1265]: Oh, exactly, thank you!

**Comment by MounirDhahri** [comment]
Thank you 🙌
#squashongreen

**Comment by MounirDhahri** [comment]
btw @irina-sid I think you missed to check the items on the PR check list. This helps us know that a PR is ready to be reviewed

**Pull request merged**